container-core/src/main/java/ai/vespa/cloud/Zone.java

```diff
@@ -0,0 +1,61 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.cloud;
+
+import java.util.Objects;
+
+/**
+ * The zone in which a cloud deployment may be running.
+ * A zone is a combination of an environment and a region.
+ *
+ * @author bratseth
+ */
+public class Zone {
+
+    private final Environment environment;
+
+    private final String region;
+
+    public Zone(Environment environment, String region) {
+        this.environment = environment;
+        this.region = region;
+    }
+
+    public Environment environment() { return environment; }
+    public String region() { return region; }
+
+    /** Returns the string environment.region */
+    @Override
+    public String toString() { return environment + "." + region; }
```

Comment:
Perhaps this should be e.g. ´stringValue()´, and return a String that is less tempting to parse from toString.

Reply:
I would agree if this was further in but I think this is fine for an "edge API" class.